hare/eligibility/beacon_test.go

```diff
@@ -18,40 +18,12 @@ func (mbp mockBeaconProvider) GetBeacon(types.EpochID) ([]byte, error) {
 	return mbp.value, nil
 }
 
-// TODO(nkryuchkov): enable when beacon sync is finished
 func TestBeacon_Value(t *testing.T) {
-	t.Skip()
-
-	r := require.New(t)
-
-	b := NewBeacon(nil, 0, logtest.New(t))
-	c := newMockCacher()
-	b.cache = c
-
 	beaconValue := []byte{1, 2, 3, 4}
-	b.beaconGetter = &mockBeaconProvider{beaconValue}
-	b.confidenceParam = cfg.ConfidenceParam
-	val, err := b.Value(context.TODO(), 100)
-	r.NoError(err)
-	r.Equal(binary.LittleEndian.Uint32(beaconValue), val)
-	r.Equal(1, c.numGet)
-	r.Equal(1, c.numAdd)
+	encodedValue := uint32(67305985) // binary.LittleEndian.Uint32(beaconValue)
```

Comment:
I think hex number would be more readable here
```suggestion
	encodedValue := uint32(0x4030201) // binary.LittleEndian.Uint32(beaconValue)
```

Reply:
you are right. done.